ansible/roles/aspenmesh-servicemesh-setup/files/cacerts.yml

```diff
@@ -0,0 +1,12 @@
+  cert-chain.pem: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUNIakNDQWNPZ0F3SUJBZ0lDTXVjd0NnWUlLb1pJemowRUF3SXdLekVUTUJFR0ExVUVDZ3dLUVhOd1pXNGcKVFdWemFERVVNQklHQTFVRUF3d0xVbTl2ZENCRFFTQmxZMk13SGhjTk1qRXdNakkwTVRrek5ERXdXaGNOTWpNdwpNakkwTVRrek5ERXdXakJGTVJNd0VRWURWUVFLREFwQmMzQmxiaUJOWlhOb01Sd3dHZ1lEVlFRRERCTkpiblJsCmNtMWxaR2xoZEdVZ1EwRWdaV05qTVJBd0RnWURWUVFIREFkM1pXSm1kWE5sTUZrd0V3WUhLb1pJemowQ0FRWUkKS29aSXpqMERBUWNEUWdBRTR0TjZiWCt0ek9ibjNSOTIvczI2RkhNaXBWT2NuZzlPZCtMKzdVNTNzTHZuY3VvVAprb01Qc0F1TWxLb2duNjFVVWhtcmVMY0FYY2pFMXlhMHJrSnRHcU9CdkRDQnVUQWRCZ05WSFE0RUZnUVU2amZlCm1MQm1LNUdhWXpXbENrbFIwU0tFT3hzd0VnWURWUjBUQVFIL0JBZ3dCZ0VCL3dJQkFEQU9CZ05WSFE4QkFmOEUKQkFNQ0F1UXdkQVlEVlIwUkJHMHdhNFl4YzNCcFptWmxPaTh2WTJ4MWMzUmxjaTVzYjJOaGJDOXVjeTlwYzNScApieTF6ZVhOMFpXMHZjMkV2WTJsMFlXUmxiSVlyYzNCcFptWmxPaTh2ZDJWaVpuVnpaUzl1Y3k5cGMzUnBieTF6CmVYTjBaVzB2YzJFdlkybDBZV1JsYklJSmJHOWpZV3hvYjNOME1Bb0dDQ3FHU000OUJBTUNBMGtBTUVZQ0lRQ0YKMjF3emtxRWRUN2RWQllVVlRzamtZdndxMjQxdERZREpnV0NjemFZZHRRSWhBSzNaQ1poR3A4Wng0ckVWUmdHQQp3L1FFb1ZzZ1FFWUY5aTh5bnVRdTllZC8KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQotLS0tLUJFR0lOIENFUlRJRklDQVRFLS0tLS0KTUlJQm1UQ0NBVUNnQXdJQkFnSVVXTUhBaHRqdk5iSGlwbDRORE9zMFFUSVpWSGd3Q2dZSUtvWkl6ajBFQXdJdwpLekVUTUJFR0ExVUVDZ3dLUVhOd1pXNGdUV1Z6YURFVU1CSUdBMVVFQXd3TFVtOXZkQ0JEUVNCbFkyTXdIaGNOCk1qRXdNakkwTVRrek16VTVXaGNOTXpFd01qSXlNVGt6TXpVNVdqQXJNUk13RVFZRFZRUUtEQXBCYzNCbGJpQk4KWlhOb01SUXdFZ1lEVlFRRERBdFNiMjkwSUVOQklHVmpZekJaTUJNR0J5cUdTTTQ5QWdFR0NDcUdTTTQ5QXdFSApBMElBQkRLcUx5aFZSQ2dGK1JtSjJNTitEQWNyS2huVEo3ZzczN0pQQldwV2R5UHhmOCtvd3RzZXZOd2YrVm5TCjRhZFduTnpiSlhjV1VySUxZMGRtSDcva05PU2pRakJBTUIwR0ExVWREZ1FXQkJUMXVtTnJBcitJQWx1ZGhBdWkKMHVUdU5tSjZyREFQQmdOVkhSTUJBZjhFQlRBREFRSC9NQTRHQTFVZER3RUIvd1FFQXdJQzVEQUtCZ2dxaGtqTwpQUVFEQWdOSEFEQkVBaUJEclRiS2JkNEg1ejN0K1crZ20xdGsyeXJLWHZYeng1TEQ0dVpPK1ova1N3SWdDekVTCmlLd2QrczQ1UkQxNmtsVVBOeTh2eE9GMUNFam8yN2RIOGppR0RHMD0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
+  root-cert.pem: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJtVENDQVVDZ0F3SUJBZ0lVV01IQWh0anZOYkhpcGw0TkRPczBRVElaVkhnd0NnWUlLb1pJemowRUF3SXcKS3pFVE1CRUdBMVVFQ2d3S1FYTndaVzRnVFdWemFERVVNQklHQTFVRUF3d0xVbTl2ZENCRFFTQmxZMk13SGhjTgpNakV3TWpJME1Ua3pNelU1V2hjTk16RXdNakl5TVRrek16VTVXakFyTVJNd0VRWURWUVFLREFwQmMzQmxiaUJOClpYTm9NUlF3RWdZRFZRUUREQXRTYjI5MElFTkJJR1ZqWXpCWk1CTUdCeXFHU000OUFnRUdDQ3FHU000OUF3RUgKQTBJQUJES3FMeWhWUkNnRitSbUoyTU4rREFjcktoblRKN2c3MzdKUEJXcFdkeVB4Zjgrb3d0c2V2TndmK1ZuUwo0YWRXbk56YkpYY1dVcklMWTBkbUg3L2tOT1NqUWpCQU1CMEdBMVVkRGdRV0JCVDF1bU5yQXIrSUFsdWRoQXVpCjB1VHVObUo2ckRBUEJnTlZIUk1CQWY4RUJUQURBUUgvTUE0R0ExVWREd0VCL3dRRUF3SUM1REFLQmdncWhrak8KUFFRREFnTkhBREJFQWlCRHJUYktiZDRINXozdCtXK2dtMXRrMnlyS1h2WHp4NUxENHVaTytaL2tTd0lnQ3pFUwppS3dkK3M0NVJEMTZrbFVQTnk4dnhPRjFDRWpvMjdkSDhqaUdERzA9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
```

Comment:
FYI, these are all self signed sample CA files, required for aspenmesh deployment.